modules/sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-subscribing-to-the-operators-needed-for-platform-configuration_{context}"]
+= Subscribing to the Operators
+
+The subscription provides the location to download the Operators needed for platform configuration.
+
+.Procedure
+
+* Use the following example to configure the subscription:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  channel: "stable" <1>
+  name: cluster-logging
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+---
```

Comment:
Add a line to each subscription:
  installPlanApproval: Manual

Add a callout for this which says:
The installPlanApproval may be set to Manual or Automatic. In Automatic mode the operator will automatically update to the latest version in the channel as they become available in the registry. In Manual mode new operator versions will be installed only after explicitly approved by the user.

Reply:
Stephen to Add under `spec` at same level as channel.